### PR TITLE
Move product versions and build numbers to _config.yml

### PR DESCRIPTION
**Pull request by sbesson**
This PR proposes to migrate the management of the product versions to the top-level `_config.yml` to ease the maintenance, updates and prevent out-of-sync issues.

The first commit leaves the artifact file sizes hardcoded in the product downloads pages for now. Happy to hear thoughts on how to deal with those. They could also be migrated to the top-level configuration file as well if we want to manage everything  that way.

### CODE REVIEW

File: products/ome-files/downloads/index.html [hunk 1]
```diff
@@ -100,8 +100,8 @@ <h3 class="text-center">Self-contained "super" builds</h3>
                     <div class="card-section">
                         <img alt="Windows logo" src="{{ site.baseurl }}/img/logos/3rd-party_windows.png">
                         <h5>VC14 32‑bit (VS2015)</h5>
```
Comment (hflynn) [inline products/ome-files/downloads/index.html]: While you're editing this anyway, 'Windows' is missing from the beginning of this heading

**Comment by hflynn** [comment]
I'm not sure re: download sizes - that's quite a lot of extra parameters to add to the config file but on the other hand, they are easier to find there than scrolling through the html on the download page. So if we are keeping them, it might be worth migrating them.

If we move to manually updating the link to the release announcement being the only thing we need to update on the download page, that's at least near the top and easy to find.

**Comment by sbesson** [comment]
Pushed the missing Windows change.

Re  artifacts sizes, I propose to postpone the migration to later. Part of the question is related to the maintenance of these values. Ideally it should be possible to either reuse/extend [ome-release](https://github.com/openmicroscopy/ome-release) and/or write a local Python script generating a list of human-readable sizes using e.g. `urllib2` as in:

```
>>> r=urllib2.urlopen("http://downloads.openmicroscopy.org/bio-formats/5.2.4/artifacts/bioformats_package.jar")
>>> float(r.headers['content-length'])/1000/1000
30.163378
>>> float(r.headers['content-length'])/1024/1024
28.76603889465332
```

**Comment by sbesson** [comment]
As this has been approved and is conflicting with #69, merging so that we can move forward. As mentioned above the maintenance of download sizes will need to happen separately.

**Pull request merged**